src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -413,20 +421,21 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, OrgPersonMix
   @Getter isFirmChangeFiling!: boolean
   @Getter isFirmConversionFiling!: boolean
   @Getter isFirmCorrectionFiling!: boolean
+  @Getter isLimitedExtendRestorationFiling!: boolean
+  @Getter isLimitedConversionRestorationFiling!: boolean
   @Getter isRoleStaff!: boolean
   @Getter hideChangeButtonForSoleProps!: boolean
 
   /** V-model for dropdown menus. */
   protected dropdown: Array<boolean> = []
 
   /** Headers for the person table. */
-  get tableHeaders () {
-    return [
-      'Name',
-      'Mailing Address',
-      'Delivery Address',
-      this.isBenBcCccUlcCorrectionFiling ? 'Roles' : ''
-    ]
+  get tableHeaders (): Array<string> {
+    const headers = ['Name', 'Mailing Address']
+    if (this.showDeliveryAddressColumn) headers.push('Delivery Address')
+    if (this.showRolesColumn) headers.push('Roles')
+    if (this.showEmailColumn) headers.push('Email Address')
+    return headers
   }
 
```

Comment:
Show only applicable columns

Reply:
It looks like the 4 columns above are equal widths -- but the actions column should be narrower (fixed width).

Reply:
Fixed width for actions column

![image](https://user-images.githubusercontent.com/116035339/224370573-04c3ac16-c841-4c1d-9474-0692c40ac1e4.png)

Reply:
Did you test ChangeSummary (change of registration filing) and ConversionSummary (record conversion filing) to verify that they display correctly?

Reply:
Yes. screenshots to compare from DEV and my LOCAL.

Change Filing
DEV:

![image](https://user-images.githubusercontent.com/116035339/224385480-fb39b1ed-10b9-4df7-a2db-939a36f84cdb.png)

 LOCAL:

![image](https://user-images.githubusercontent.com/116035339/224385652-9bb9b607-36ae-4497-9536-851eb092411f.png)


Correction Filing
DEV:

![image](https://user-images.githubusercontent.com/116035339/224385774-97768433-16cb-47af-b5fe-89db120928a9.png)

LOCAL:

![image](https://user-images.githubusercontent.com/116035339/224385852-14114914-4cc6-4a4f-8b0a-869e64c0498a.png)

Reply:
Awesome! Good job. Please add a "Test Notes" comment to your ticket to call this out as something you changed but should look the same (for UX and QA to verify).

PS Record Conversion is not the same as Correction, but the screenshots are correct 👍

Reply:
You can resolve this conversation if you like. I'm satisfied 😀

Reply:
sounds good! thanks